dvc/repo/params/show.py

```diff
@@ -117,8 +119,8 @@ def show(repo, revs=None, targets=None, deps=False, onerror: Callable = None):
     # Hide workspace params if they are the same as in the active branch
     try:
         active_branch = repo.scm.active_branch()
-    except (TypeError, NoSCMError):
-        # TypeError - detached head
+    except (SCMError, NoSCMError):
```

Comment:
I think this may still get raised on a `gitpython` backend, which probably should be handled upstream.

Reply:
You have this construction in `metrics.show()` too, and it doesn't have `TypeError` there.

Reply:
BTW, maybe moving this to commands code will improve code - this feels like a presentation logic.